LibertyDSNP/example-client
# Feature/profile preview in edit registration/179585383

Purpose
---------------
Feature
[https://www.pivotaltracker.com/story/show/179585383](https://www.pivotaltracker.com/story/show/179585383)
As a user, I want to see a preview of my profile when selecting handles.

Solution
---------------
- Add profile preview in the editRegistration component
- style after the above wireframes in zeplin

Change summary
---------------
* created RegistrationPreview component
* styled accordingly
* passed the user info to preview into that component

Steps to Verify
----------------
1. try logging in and selecting a handle.
2. see your profile preview


Additional details / screenshot
---------------

https://user-images.githubusercontent.com/43625033/134993890-b9aa9fc0-2eff-4955-b54a-ed8059747937.mov
